docs/developer/creating-test-environment.md

```diff
@@ -20,29 +20,25 @@ You can choose between either a single-cluster or a two-cluster setup.
 
 To bundle your module image and operator, please refer to the detailed information inside [template-operator docs](../../samples/template-operator/README.md#bundling-and-installation).
 
-### Install Kyma and run lifecycle-manager operator
+### Create Kyma custom resource
 
 1. If you're using the two-cluster setup, set your `KUBECONFIG` to the KCP Cluster context.
-
-2. Create the `kyma-system` Namespace:
-
-   ```sh
-   kubectl create ns kyma-system
-   ```
-
-3. Create a Secret to access the cluster which acts as SKR:
+   Next, create a `Secret` to access the SKR cluster from KCP:
 
    Go to `https://github.com/kyma-project/lifecycle-manager` and run the following commands, to create a secret and apply it to the KCP Cluster:
 
    ```sh
    chmod 755 ./operator/config/samples/secret/k3d-secret-gen.sh
    ./operator/config/samples/secret/k3d-secret-gen.sh
    ```
-   > _**NOTE:**_ In the **single-cluster setup**, adjust your contexts for applying the secret using `KCP_CLUSTER_CTX` and `SKR_CLUSTER_CTX` - both should point to the same cluster.
```

Comment:
I think the note could stay but without saying they should point to the same cluster no?

Reply:
In single cluster mode, the secret is not required  :)